bfi_lib/src/persistence.rs

```diff
@@ -54,3 +54,69 @@ impl ExtractedBfiData {
         Ok(())
     }
 }
+
+/**
+ * Parquet conversion of extracted BFI data from live capture
+ */
+impl SinglePacketBfiData {
```

Comment:
I dont think this is a good idea. Concatenating dataframes and writing them to disk is quite expensive as it stands already. We should always opt to keeping larger buffers of data in memory and then writing them out as a fully assembled dataframe as rarely as possible.

I suggest doing the following:

- Remove to_parquet for SinglePacketBfiData
- Allow concatenation in to_parquet of ExtractedBfiData
- In the live capture, once a batch reaches "critical" size: Convert the batch of SinglePacketBfiData to ExtractedBfiData (as we did already in the file version), and simply use the to_parquet function we already have.